### ISSUE TITLE
[Ubuntu Server] Linking terms to glossary

### ISSUE THREAD

**Issue by s-makin** [canonical/open-documentation-academy#235]
## Task

Now that the Ubuntu Server docs [have a glossary](https://github.com/canonical/ubuntu-server-documentation/pull/124), and our contributors have been hard at work [defining the terms](https://github.com/canonical/open-documentation-academy/issues/166), it would be awesome to start connecting more of the terms to the glossary to help our readers :D

## Details

This issue has been created specifically for the **UbuCon CODA workshop** and will be closed after the event finishes. It is suitable for people making their first documentation contributions.

Unlike the majority of our tasks, this task can be worked on by more than one person at a time. You do not need to be the sole assignee to work on this task, but we do ask that you leave a comment to say which page you're working on to help others who may be interested.

For this issue, you can use the GitHub web interface to submit your changes. No need to use the terminal or spending a long time setting up!

## What should I do?

* Pick a page -- any page you like! -- in the [Server documentation](https://documentation.ubuntu.com/server/). I'll call this "your page" from here on.
* Read through your page, and if you spot a piece of jargon or a technical term you don't understand, or see an acronym, check if the term already exists [in the glossary page](https://documentation.ubuntu.com/server/reference/glossary/)

### If the term is in the glossary already

Click on the "edit this page" link at the bottom of the page you are reading.

Then, Link the term to the glossary by tagging it in a `{term}` role:

```
{term}`the terminology in the text <glossary term>`
```

Where `<glossary term>` needs to match (including capitalization) exactly with the term as it appears in the glossary list. Here's an example. Let's imagine on your page that the acronym "API" is used without being defined yet. You should change the text on your page to be like this:
```
{term}`API`
```
or, a better way would be to expand the acronym on your page, too.
```
{term}`Application Programming Interface (API) <API>`
```
In either case, "API" is the term in the glossary list, and it acts as a reference point so that Sphinx looks up the term "API" and the reader will automatically be shown the corresponding definition when they hover over it. You can see how this works [on this page](https://documentation.ubuntu.com/server/how-to/networking/install-dns/).

**Note:** If the term appears more than once on the page, you only need to tag it **once** (usually the first time it appears)

When you're done suggesting changes to your page, you can submit them as a pull request.
 
### If it's NOT in the glossary

If the terminology *is not* in the glossary, but it confused you, note it down!

Then, when you're done with the page, report them to us by clicking the "Give feedback" button on the top of the page, and tell us all the terms you found on your page that were missing from the glossary. Using this button will automatically include the page you were working on.

## Thank you!

This will greatly help us to make our documentation as accessible as possible by providing definitions for technical terms at the place where our readers need to know them.

We very much appreciate your taking the time to help us :)

**Comment by Osike** [comment]
@s-makin I would like to work on this task

**Comment by s-makin** [comment]
> [@s-makin](https://github.com/s-makin) I would like to work on this task

That's awesome, thank you! Our [Contributing guide](https://documentation.ubuntu.com/server/contributing/) should have all the information you need -- if it doesn't or you need help with anything, please let me know :)
